packages/babel-plugin-transform-async-generator-functions/test/fixtures/for-await/async-function/expected.js

```diff
@@ -1,5 +1,5 @@
 let f = (() => {
-  var ref = babelHelpers.asyncToGenerator(function* () {
+  var _ref = babelHelpers.asyncToGenerator(function* () {
```

Comment:
Not sure why this happened but some other code changed in the time between the original PR and now